pkg/logging/logging.go

```diff
@@ -0,0 +1,88 @@
+package logging
```

Comment:
At least one file in each package should contain a documentation indicating what the package does. You can see examples in other packages inside the repository.